data/pages.yaml

```diff
@@ -1,51 +1,67 @@
+# Latest Ansible package version.
+ansible_pkg:
+  announce: Ansible 7 is now available
+  pip_install: pip install ansible
+# Base docsite page names.
+site:
+  index: index.html
+  user: user.html
+  developer: developer.html
+  maintainer: maintainer.html
+  collections: collections.html
+  community: community.html
+  core: core.html
+  ecosystem: ecosystem.html
+# Content for the index page.
 index:
   welcome: Welcome to Ansible documentation
   about_ansible: Ansible offers powerful and flexible automation that is simple, decentralized, and easy.
-  #Ansible novice
+  # Ansible novice
   novice_actions:
-    #Labels provide meaningful descriptions of elements for accessibility purposes.
-    action_label: Button group that links to documentation for new Ansible users.
-    action_one: Understand the fundamentals of Ansible automation
-    action_two: Set up an Ansible automation project
-    action_three: Write your first playbook in a few easy steps
-  #Ansible installation
-  install_action: Go to the installation guide
-  #Ansible user
+    # Labels provide meaningful descriptions of elements for accessibility purposes.
+    label: Button group that links to documentation for new Ansible users.
+    learn: Understand the fundamentals of Ansible automation
+    setup: Set up an Ansible automation project
```

Comment:
```suggestion
    setup: Install the Ansible package
```

Reply:
The original text doesn't feel right to me. This is specifically only dealing with ansible package install, not setting up an automation project as a whole. When I see the original text, I think it's doing something more (and potentially scary as a new user).

Reply:
Yeah I think I put "set up automation project" as a gap but didn't alter the text here: https://github.com/ansible/jinja-docsite/issues/32